src/editing.md

```diff
@@ -180,31 +182,82 @@ this is not recommended, for the following reasons:
   trees with thousands of items can actually break the display of
   the deck list in the current implementation.
 
+### Using Tags
+
 Instead of creating lots of little decks, it's a better idea to use tags
-and/or fields to classify your content. Instead of creating a "food
-verbs" decks for example, you could add those cards to your main
-language study deck, and tag the cards with "food" and "verb". Each card
-can have multiple tags, which means you can do things like search for
-all verbs, or all food-related vocabulary, or all verbs that are related
-to food.
+and/or fields to classify your content. Tags are a useful way to boost 
+search results, find specific content, and maintain your collection 
+organized, in addition to (or in combination with) your deck / subdecks 
+structure. There are many ways of using tags and flags effectively, and
+thinking in advance about how you want to use them will help you decide
+what will work best for you.
+
+Some people prefer using decks and subdecks to keep their cards organized,
+but using tags have a big advantage over decks for that: you can add several
+tags to a single note, but a single note can only belong to one deck. This 
+allows using [filtered decks](filtered-decks.md), and makes tags a more 
+powerful and flexible categorization system than decks in most cases. 
```

Comment:
Filtered decks can be used with decks as well, so this is not really "selling" the use of tags. Recommend you cut this sentence or reword it.